usaspending_api/database_scripts/job_archive/management/commands/generate_cares_act_test_data_helpers.py

```diff
@@ -0,0 +1,265 @@
+"""
```

Comment:
minor, but this is probably better refactored into a folder, lest someone try to run `manage.py generate_cares_act_test_data_helpers`

Reply:
You don't have a lot of faith in us do you.